core/lens/lens_list.mli

```diff
@@ -0,0 +1,22 @@
+open Utility
+
```

Comment:
As per the above comment, I think this module is unnecessary.

Reply:
See above comment.